src/pages/invite.js

```diff
@@ -15,16 +15,20 @@ import TwitterIcon from '../components/Atoms/TwitterIcon'
 import {LayoutColumn2, LayoutBottom, TextFieldsWrapper} from '../components/styled'
 import {withAuthStatusContext, withRequestContext} from '../context/HOC'
 import { navigateTo } from 'gatsby-link';
+import cloudinary from '../utils/cloudinary';
```

Comment:
Directly importing this without using React Context, as no other component uses Cloudinary at the moment.